i18n/ru.toml

```diff
@@ -0,0 +1,60 @@
+[home]
+other = "Главная"
+
+[backToHome]
+other = "На главную"
+
+[flip]
+other = "Повернуть!"
+
+[archives]
+other = "Архивы"
+
+[allTags]
+other = "Все теги"
+
+[tag]
+one = "1 тег"
+few = "{{ .Count }} тега"
+many = "{{ .Count }} тегов"
+other = "{{ .Count }} тегов"
+
+[allCategories]
+other = "Все категории"
+
+[category]
+one = "1 категория"
+few = "{{ .Count }} категории"
```

Comment:
few and many are not needed.

Reply:
Unfortunately I can't build a fine site without `many`. The Hugo logs warnings with "ru" settings.

Reply:
I have not encountered this situation 🤔, but it's safe to remain them. Yes, we can clear the `few`.

Reply:
Sorry

```shell
Change detected, rebuilding site.
2021-05-12 13:19:10.422 +0300
i18n changed "/raid/user/github.com/contributorpw/hugo-theme-dream/i18n/ru.toml": WRITE
WARN 2021/05/12 13:19:10 Failed to get translated string for language "ru" and ID "article": message "article" has no plural form "few"
WARN 2021/05/12 13:19:10 Failed to get translated string for language "ru" and ID "article": message "article" has no plural form "few"
WARN 2021/05/12 13:19:10 Failed to get translated string for language "ru" and ID "article": message "article" has no plural form "few"
Total in 46 ms
```